SPECS/kernel/config

```diff
@@ -507,7 +507,6 @@ CONFIG_PHYSICAL_ALIGN=0x1000000
 CONFIG_DYNAMIC_MEMORY_LAYOUT=y
 CONFIG_RANDOMIZE_MEMORY=y
 CONFIG_RANDOMIZE_MEMORY_PHYSICAL_PADDING=0xa
-# CONFIG_ADDRESS_MASKING is not set
```

Comment:
Result of: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/60a5ba560f296ad8da153f6ad3f70030bfa3958f